README.md

```diff
@@ -266,6 +266,7 @@ Databases and data stores with C APIs.
   any desired width. Has a SIMD-optimized implementation. [``WTFPL``][WTFPL]
 * [uthash][273] - Single-file hash table implementation. [``BSD-1-Clause``][BSD-1-Clause]
 * [vector.h][152] - Header library for typed lists. [``MIT``][MIT]
+* [libds](https://github.com/recp/ds) - Common Data Structures and Algorithms. [MIT](https://github.com/recp/ds/blob/master/LICENSE)
```

Comment:
Please don't use inline links - see the other entries for examples of how to do this. Additionally, I don't think your contribution meets the standards required of the entries on here, specifically because it lacks documentation, and its README is full of grammatical errors, which suggests that it's rushed or under-thought.